src/tasks/installers/common-tasks.ts

```diff
@@ -234,3 +235,17 @@ export function getPrintHighlightedMessagesTask(): Listr.ListrTask {
     }
   }
 }
+
+export function isOlmPreInstalledTask(kube: KubeHelper): Listr.ListrTask<Listr.ListrContext> {
```

Comment:
I would call it `ensureOlmPreInstalledTask` as it doesn't return a value, but terminates execution instead.